<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>Модель</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Параметры</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="16" column="1">
           <widget class="QTableWidget" name="tableWidget_2"/>
          </item>
          <item row="1" column="0">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="8" column="0">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="6" column="0">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Количество  нейронов (n)</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Количество слоев</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Ограничение (В)</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>М2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Шаг градиентного спуска</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>М1</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLineEdit" name="lineEdit_7"/>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Точность</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Время (T)</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_8"/>
          </item>
          <item row="16" column="0">
           <widget class="QTableWidget" name="tableWidget">
            <column>
             <property name="text">
              <string>xi(0)</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Ai</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Gi</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="11" column="0" colspan="2">
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>PushButton</string>
            </property>
           </widget>
          </item>
          <item row="15" column="1">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Управление</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0" rowspan="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Иксы</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Таблицы</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Графики</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Страница</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
